draft-cenzano-moq-media-interop.md

```diff
@@ -85,6 +85,23 @@ in {{audio-object-format}}.
 TODO: Datagram forwarding preference could be used, but has problems if audio
 frame does not fit in a single UDP payload.
 
+## Timestamps
+To avoid using fractional numbers and having to deal with rounding errors,
+timestamps will be expressed with two integers:
+- timestamp numerator (ex: PTS, DTS, duration)
+- timebase
+
+To convert a timestamp into seconds you just need to:
+timestamp(s) = timestamp numerator / timebase
```

Comment:
For timestamps longer than 1s is the numerator just larger than the denominator eg: 65 / 30?